### PR TITLE
Desktop portfolio version

### PR DESCRIPTION
**Pull request by Outkarm**
optimized webpage to fit desktop screen

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -223,8 +224,8 @@
   font-family: "Inter", sans-serif;
   font-style: normal;
   font-weight: 500;
-  font-size: 15px;
-  line-height: 20px;
+  font-size: 0.938rem;
+  line-height: 1.25rem;
   color: #091e42;
   flex: none;
```
Comment (ElsonOtake) [inline style.css]: - [x] I think it's better to use `flex-basis=content` to fix the alignment problem of this list. Your portfolio will look more professional.

File: style.css [hunk 2]
```diff
@@ -239,7 +240,7 @@
   align-items: center;
   padding: 12px;
   gap: 10px;
-  width: 124px;
+  width: 33.067%;
   height: 48px;
   background: #ff6b00;
   flex: none;
```
Comment (ElsonOtake) [inline style.css]: - [ ] I think it's better not to use `flex` for this button that is causing alignment problems. Your portfolio will look more professional.

File: style.css [hunk 3]
```diff
@@ -248,39 +249,62 @@
...
+  font-size: 1rem;
+  line-height: 1.5rem;
   text-align: center;
   letter-spacing: 0.03em;
   color: #fff;
   border: none;
 }
 
-.various-projects {
+.various-projects-1,
```
Comment (ElsonOtake) [inline style.css:236]: - [ ] Please add a border to this project. It's a project requirement to follow the template in Figma.

File: style.css [hunk 4]
```diff
@@ -688,3 +739,587 @@
...
+    font-size: 2rem;
+    line-height: 2.25rem;
+    margin-bottom: 32px;
+    color: #fff;
+  }
+
+  .semi-list {
+    list-style: none;
+    text-decoration: none;
+    text-align: center;
```
Comment (ElsonOtake) [inline style.css]: - [ ] I think it's better to align this list to the left following the Figma template. It's a project requirement.

File: style.css [hunk 5]
```diff
@@ -688,3 +739,587 @@
...
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding: 0;
+    list-style: none;
+  }
+
+  #name {
+    margin: 15px 16px;
+    width: 330px;
```
Comment (ElsonOtake) [inline style.css:1139]: - [ ] I think it's better to use the same width of the `<textarea>`. Will be more suitable for the Figma template.

File: style.css [hunk 6]
```diff
@@ -688,3 +739,587 @@
...
+    font-style: normal;
+    font-weight: 400;
+    font-size: 0.938rem;
+    line-height: 1.25rem;
+    color: #979493;
+  }
+
+  #mail {
+    margin: 15px 16px;
+    width: 330px;
```
Comment (ElsonOtake) [inline style.css:1163]: - [ ] I think it's better to use the same width of the `<textarea>`. Will be more suitable for the Figma template.

File: style.css [hunk 7]
```diff
@@ -4,32 +4,40 @@
   box-sizing: border-box;
 }
```
Comment (ElsonOtake) [inline style.css:6]: - [ ] I think it's better to check the behavior of your portfolio visualization at transitions from mobile to desktop, and vice-versa. Check that using the [Chrome Inspect](https://www.youtube.com/watch?v=uInIQJJIzHE).
Your portfolio is presenting some undesirable look:
button without space to line
Screen Shot 2022-08-26 at 11 31 07 AM

Small input fields
Screen Shot 2022-08-26 at 11 33 10 AM

Content outside the screen
Screen Shot 2022-08-26 at 11 31 34 AM

**Review by ElsonOtake — CHANGES_REQUESTED** [review]
STATUS: CHANGES REQUIRED 🟡 

Hi @Outkarm,

Good job so far! 🎉 🎉 
Nice work!! 🏆 🏆 
There are some issues that you still need to work on to go to the next project but you are almost there! 🥲 🥲 

woohoo-youre-almost

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Professional README file. ✔️

## Required Changes ♻️
_Check the comments under the review._

- [x] Please write a meaningful title for your pull request and always add a short summary of changes introduced in the pull request. More informations in skills/blob/main/documentation/clean_repo.md)
- [x] Please write your commit message in the imperative tense and capitalized. More informations in skills/blob/main/documentation/clean_repo.md)

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (Outkarm) [reply style.css]: done

File: style.css (hunk 2 above)
Comment (Outkarm) [reply style.css]: done

File: style.css (hunk 3 above)
Comment (Outkarm) [reply style.css:236]: please explain further

File: style.css (hunk 4 above)
Comment (Outkarm) [reply style.css]: done

File: index.html [hunk 8]
```diff
@@ -250,90 +268,96 @@ <h3 class="about-heading">About me</h3>
...
+            <li>
+              <label for="name" class="name-section"></label
+              ><input
+                type="text"
+                name="user_name"
+                id="name"
+                placeholder="Full name"
+                maxlength="30"
+                required
+              />
```
Comment (HAMMAS-SALEEM) [inline index.html:331]: ### Your Form
your-design

### Figma Form
Screenshot from 2022-08-28 03-06-39

- [ ] You can see that the Figma form is expanding on the whole screen in mobile version but your fields are not as the previous reviewer suggested. Please make it look like the figma template :blush: to make your project look professional :+1:

**Hint: Remove the widths of input fields in the mobile version then your input fields will expand and look like the Figma template :blush:**

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
almost-there

### Hey @Outkarm :wave:

### Highlights :1st_place_medal:
 
:heavy_check_mark: Meaningful commits
:heavy_check_mark: Added .gitignore
:heavy_check_mark: Following GitFlow
:heavy_check_mark: Descriptive PR Title
:heavy_check_mark: Following Linters

Good job on implementing some of the changes requested by the previous reviewer, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional Suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @HAMMAS-SALEEM in your question so I can receive the notification. Otherwise reach out to me on slack which is more fast. You can reach out to me on slack by searching my name in **full-time-students** channel :+1:

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 8 above)
Comment (Outkarm) [reply index.html:331]: i have just done that

Comment (Outkarm) [reply index.html:331]: thanks for pointing it out

**Review by ichala — APPROVED** [review]
Hi @Outkarm ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  

 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Good Design ✔️
- No linter Errors✔️


 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
*************************************
# Optional Suggestions 

- [ ]  N/A



_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  @ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._v

**Pull request merged**